src/cenclave_lib_sgx/cli.py

```diff
@@ -241,6 +243,12 @@ def run() -> None:
         "wsgi_max_body_size": 2 * 1024 * 1024 * 1024,  # 2 GB
     }
 
+    if client_cert := args.client_certificate:
+        config_map["verify_mode"] = 2
```

Comment:
declare a const for this `2` value? (for readability I mean)

Reply:
Fixed in 55574cb89cdbb4fcb7a7e3cce7cfd59baf7f3f45.